source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -7124,8 +6943,18 @@ export class Authentication extends SerializableObject {
     tokenExchangeResource?: TokenExchangeResource;
 }
 
+export interface ISingletonContainer {
```

Comment:
nit: can you consider adding comments or re-naming Singleton? In the context of programming, the first thing that comes to mind is the singleton design pattern; not the singleton set.

Reply:
happy to entertain other naming ideas -- I just picked the first thing that came to mind, and haven't been able to come up with something better :)